roles/docker/handlers/main.yml

```diff
@@ -2,12 +2,18 @@
 - name: restart docker
   command: /bin/true
   notify:
+    - Docker | clean up orphans
     - Docker | reload systemd
     - Docker | reload docker.socket
     - Docker | reload docker
     - Docker | pause while Docker restarts
     - Docker | wait for docker
 
+- name: Docker | clean up orphans
+  script: clean-up-orphans.sh
```

Comment:
Please use `command`

Reply:
If we use command, then we will have to add an extra step of copying the script to remote nodes.